lib/matplotlib/image.py

```diff
@@ -533,7 +533,7 @@ def _make_image(self, A, in_bbox, out_bbox, clip_bbox, magnification=1.0,
                 # that may have moved input values in/out of range
                 s_vmin, s_vmax = vrange
                 if isinstance(self.norm, mcolors.LogNorm):
-                    if s_vmin < 0:
+                    if s_vmin <= 0:
                         s_vmin = max(s_vmin, np.finfo(scaled_dtype).eps)
```

Comment:
Why is there a `max` here? `.eps` should be greater than  `s_vmin` due to the condition, so no need for `max`, or else drop the `if` instead.

Reply:
I agree with that.